BPOR/BPOR.Rms/Views/Shared/_Layout.cshtml

```diff
@@ -6,7 +6,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta name="x-environment" content="@HostEnvironment.EnvironmentName" />
-    <title>@ViewData["Title"] - Be Part of Research Volunteer Registry</title>
+    <title>@ViewData["Title"] - Be Part of Research volunteer registry - Be Part of Research</title>
```

Comment:
Is this what the client has requested?

It seems redundant to duplicate `Be Part of Research` in the title.

What is the reasoning behind this title format?

Reply:
this is the format that was requested, I am not aware of the specific reasoning, but I believe they are following the format of this example https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G88#G88-ex1 from wcag 2.0